### ISSUE TITLE
Cannot read property 'length' of null   Code: -32603

### ISSUE THREAD

**Issue by rahularyan** [HvyIndustries/crane#234]
Hello,

Thanks for creating such useful extension. Just now updated to latest version and getting this error

```
Request textDocument/completion failed.
    Message: Request textDocument/completion failed with message: 
    Cannot read property 'length' of null Code: -32603 
```

This error automatically show when I press `>`.
Please check and fix this issue.
Thanks

![error](https://cloud.githubusercontent.com/assets/916764/23601242/81e6e1d2-0270-11e7-898b-e0ca0932c43b.gif)

**Comment by nevadascout** [comment]
Oooh, thanks for reporting this. I'll get it fixed and released ASAP!

**Comment by nevadascout** [comment]
This is now fixed in v0.3.1

**Comment by rahularyan** [comment]
Thanks, its fine now.

**Comment by rahularyan** [comment]
Hello @nevadascout 
This issue appeared again after last update.

```[Error - 8:08:16 AM] Request textDocument/completion failed.
  Message: Request textDocument/completion failed with message: 
    Cannot read property 'label' of null
  Code: -32603 ```

**Comment by nevadascout** [comment]
Reopening this to investigate

**Comment by Basquens** [comment]
Having same issue, there is a way to at least stop Output window poping up when it occurs?

**Comment by nevadascout** [comment]
@Basquens Other than fixing the bug, I'm afraid not!

**I'm planning to release a fix for this within the next 6 hours.**

**Comment by nevadascout** [comment]
Released in v0.3.5
